CMSgov/mint-app
# [EASI-3774] Fast Follow -- Adding in missing Read Only Label

# EASI-3774

## Changes and Description

- Replaced label with the correct label

| Before      | After |
| ------ | ------ |
|![image](https://github.com/CMSgov/mint-app/assets/2349702/e462a23a-932d-481a-a77d-4530a2212bae)|![image](https://github.com/CMSgov/mint-app/assets/2349702/156993c4-f8ad-4548-bd96-7dd7d5e856f3)|





## How to test this change

1. Go to Gen Char read only page
2. check to see label is correct


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
